GMRScripts/normalise_csv.py

```diff
@@ -6,15 +6,15 @@
 from shutil import copy
 
 
-def ReadInPwr(file_name):
+def ReadInPwr(dir_name):
     '''
     Read in and process the power spectrum data file
     Args:
         file_name, string file path
     '''
-    pwr_spec = os.path.join(file_name, 'power_spectrum.csv')
+    pwr_spec = os.path.join(dir_name, 'power_spectrum.csv')
     step, wl, f, power = np.genfromtxt(pwr_spec,
-                                       delimiter=',',
+                                       delimiter='\t',

```

Comment:
power_spectrum.csv is now comma delimited.
```suggestion
     step, wl, f, power = np.genfromtxt(pwr_spec, delimiter=',', skip_header=1, unpack=True)
```